corehq/apps/accounting/models.py

```diff
@@ -1016,7 +1016,8 @@ class Subscription(models.Model):
     is_active = models.BooleanField(default=False)
     do_not_invoice = models.BooleanField(default=False)
     no_invoice_reason = models.CharField(blank=True, null=True, max_length=256)
-    do_not_email = models.BooleanField(default=False)
+    do_not_email_invoice = models.BooleanField(default=False)
+    do_not_email_reminder = models.BooleanField(default=False)
```

Comment:
i don't think it's worth it to change now, but it's generally better to name boolean properties in the positive (i.e., `send_email_invoice` and `send_email_reminder`)